tests/nemo_text_processing/en/data_text_normalization/test_cases_electronic.txt

```diff
@@ -1,20 +1,27 @@
-a.bc@gmail.com~a dot b c at G mail dot com
-a@hotmail.de~a at hotmail dot de
-a@hotmail.fr~a at hotmail dot fr
-a@hotmail.it~a at hotmail dot it
-a@aol.it~a at a o l dot it
-a@msn.it~a at m s n dot it
-cdf@abc.edu~c d f at a b c dot e d u
-abc@gmail.abc~a b c at G mail dot a b c
-abc@abc.com~a b c at a b c dot com
-asdf123@abc.com~a s d f one two three at a b c dot com
-a1b2@abc.com~a one b two at a b c dot com
-ab3.sdd.3@gmail.com~a b three dot s d d dot three at G mail dot com
-email is abc1@gmail.com~email is a b c one at G mail dot com
-test .b@gmail.com~test . b at G mail dot com
-abs@nvidia.com~a b s at nvidia dot com
-email is a-1.b3_c&-d@gma4i-l.com test~email is a dash one dot b three underscore c ampersand dash d at g m a four i dash l dot com test
+a.bc@gmail.com~a dot bc at gmail dot com
+a@hotmail.de~a at hotmail dot DE
+a@hotmail.fr~a at hotmail dot FR
+a@hotmail.it~a at hotmail dot IT
+a@aol.it~a at aol dot IT
+a@msn.it~a at msn dot IT
+cdf@abc.edu~cdf at abc dot EDU
+abc@gmail.abc~abc at gmail dot ABC
+abc@abc.com~abc at abc dot com
+asdf123@abc.com~asdf one two three at abc dot com
+a1b2@abc.com~a one b two at abc dot com
+ab3.sdd.3@gmail.com~ab three dot sdd dot three at gmail dot com
+email is abc1@gmail.com~email is abc one at gmail dot com
+test .b@gmail.com~test . b at gmail dot com
+abs@nvidia.com~abs at nvidia dot com
+email is a-1.b3_c&-d@gma4i-l.com test~email is a dash one dot b three underscore c ampersand dash d at gma four i dash l dot com test
 nvidia.com~nvidia dot com
-test.com~t e s t dot com
-http://www.ourdailynews.com.sm~h t t p colon slash slash w w w dot o u r d a i l y n e w s dot com dot s m
-sdf@gmail.com.sm~s d f at G mail dot com dot s m
+test.com~test dot com
+http://www.ourdailynews.com.sm~HTTP semicolon slash slash WWW dot ourdailynews dot com dot SM
+https://www.ourdailynews.com.sm~HTTPS semicolon slash slash WWW dot ourdailynews dot com dot SM
+www.ourdailynews.com.sm~WWW dot ourdailynews dot com dot SM
+www.ourdailynews.com/123-sm~WWW dot ourdailynews dot com slash one two three dash SM
+sdf@gmail.com.sm~sdf at gmail dot com dot SM
+sdf@gmail.com/123-sm~sdf at gmail dot com slash one two three dash SM
+sdf@gmail.abc/123-sm~sdf at gmail dot ABC slash one two three dash SM
+ourdailynews.com/123-sm~ourdailynews dot com / one hundred twenty three-sm
```

Comment:
why not `/`->slash and `-`-> dash for URL? why is it different behavior from email?

Reply:
because sdf@gmail.abc/123-sm is classified as electronic, but  ourdailynews.com/123-sm is 
tokens { electronic { domain: "ourdailynews.com" } }  tokens { name: "/" } tokens { name: "one hundred twenty three-sm" }

this is because for it to be email address we need a protocol, like www.ourdailynews.com/123-sm~